section/BlueprintFieldCheck.cpp

```diff
@@ -0,0 +1,93 @@
+#include "moho.h"
+
+const char *meshGroupField = "LODGroup";
+
+char eaxStorage[4];
+char espStorage[4];
+char ebpStorage[4];
+char edxStorage[4];
+
+char *fieldName;
+float fieldValue;
+char *rBlueprint;
+
+
+//Blueprint loading on game start. We are inside Moho::SCR_LuaBuildObject (recursive function)
+//For now it is used for "LODGroup", but additional fields can be added in the future if needed.
+ 
+void GetBpFieldValues()
+{
+	asm(
+        "mov %[eaxStorage], eax;"
+        "mov %[espStorage], esp;"
+        "mov %[ebpStorage], ebp;"
+        "mov %[edxStorage], edx;"
+        
+        "mov %[fieldName], eax;"
+        
+        "mov ecx, dword ptr [ecx+0x58];"
+        "mov %[fieldValue], ecx;"
+        
+        "mov ecx, dword ptr [esp+0x4];"
+        "mov %[rBlueprint], ecx;"
+        
+        "jmp 0x004CF7F7;"
+        :
+        : [fieldName] "m" (fieldName),
+          [fieldValue] "m" (fieldValue),
+          [rBlueprint] "m" (rBlueprint),
+          [eaxStorage] "m" (eaxStorage),
+          [espStorage] "m" (espStorage),
+          [ebpStorage] "m" (ebpStorage),
+          [edxStorage] "m" (edxStorage)
+        :
+	);
+}
+
+void ProcessBpFieldValues()
+{
+    if (not strcmp(fieldName, meshGroupField)) //(not strcmp) == equal
```

Comment:
`not` is something that isn't common for C/C++, use `!` or `== 0`.

Reply:
Yea, I just hate `!`. It's really harder to notice compare to `not`. Will change to `== 0` I think?

Reply:
For comparisons, `== 0` is the most common because the return value isn't supposed to interpreted as a boolean - it's a three-way comparison, yielding a number less than, equal to, or greater than zero. So people use `< 0`, `== 0`, and `> 0`.